src/pkg/cli/cert.go

```diff
@@ -49,21 +44,22 @@ func generateCert(ctx context.Context, domain, albDns string) {
 		return
 	}
 
-	term.Infof("CNAME record for %v is ready", domain)
+	term.Infof("%v DNS is properly configured!", domain)
 	if err := checkTLSCert(ctx, domain); err == nil {
 		term.Infof("TLS cert for %v is already ready", domain)
 		return
 	}
 	term.Infof("Triggering cert generation for %v", domain)
 
-	if err := getWithContext(ctx, fmt.Sprintf("http://%v", domain)); err != nil {
-		term.Errorf("Error triggering cert generation: %v", err)
-		return
+	if err := getWithRetries(ctx, fmt.Sprintf("http://%v", domain), 3); err != nil { // Retry incase of DNS error
+		// Ignore possible tls error as cert attachment may take time
+		term.Debugf("Error triggering cert generation: %v", err)
 	}
 
 	term.Infof("Waiting for TLS cert to be online for %v", domain)
 	if err := waitForTLS(ctx, domain); err != nil {
-		term.Errorf("Error waiting for TLS: %v", err)
+		term.Errorf("Error waiting for TLS to be online: %v", err)
+		term.Errorf("Please check for error messages from `/aws/lambda/acme-lambda` log group in cloudwatch for more details")
```

Comment:
This message only makes sense for BYOC

Reply:
This feature only works with BYOC for now, I'll add a TODO: to make sure playground receive a different message.